templates/themes/app/parts/content_fields.html.twig

```diff
@@ -42,7 +42,22 @@
                         )
                     ) }}
                 {% else %}
-                    <a href="{{ path("ibexa.url.alias", { "contentId": field.value.reference }) ~ suffix }}"{% if css_class is not empty %} class="{{ css_class }}"{% endif %}{% if field.value.isTargetLinkInNewTab %} target="_blank"{% endif %}>{{ label }}</a>
+                    {{ render(
+                        controller(
+                            "ng_content::viewAction",
```

Comment:
Shouldn'twe use `ng_content_view` here?

Reply:
We can't since we don't have Site API Content/Location here, see https://netgen.atlassian.net/browse/NGSTACK-712 for the alternative.

Reply:
:+1: